src/_includes/sidebar/default.html

```diff
@@ -1,3 +1,7 @@
 <a class="d-block{% if include.document.path == page.path %} active{% endif %}" href="{{ include.document.url }}" data-sidebar-link>
-  {{ include.document.title }}
+  {% if include.document.sidebar_title %}
```

Comment:
This could be expressed more simply as:
```
{{ include.document.sidebar_title | default: include.document.title }}
```

Reply:
Ah... good to know! Thanks! Will make the change.